blockchain/pool.go

```diff
@@ -120,9 +127,12 @@ func (pool *BlockPool) removeTimedoutPeers() {
 		if !peer.didTimeout && peer.numPending > 0 {
 			curRate := peer.recvMonitor.Status().CurRate
 			// XXX remove curRate != 0
-			if curRate != 0 && curRate < minRecvRate {
```

Comment:
im not sure why it was here in the beginning. probably something about not removing right away if the curRate is 0 and we havent received anything yet to give it time to ramp up.

Also the fast sync bash test failed here. I can't find this error message in the logs but maybe it's related? could just be non-determinism in this test :(